Package.resolved

```diff
@@ -6,7 +6,7 @@
         "repositoryURL": "https://github.com/mxcl/PromiseKit.git",
         "state": {
           "branch": null,
-          "revision": "e0176e7886b3bef5b9a546bcc6b357f6f569f222",
+          "revision": "fe1e9c5b62465227cceb7b0e6e79489ba7b824af",
```

Comment:
This is not related, please remove.

Reply:
Sorry about that, done!

Reply:
nps and thanks! Will merge and release once CI passes.